css/style.css

```diff
@@ -1487,9 +1487,22 @@ h2 a {
   max-width: 100%;
   margin: auto;
 }
-.blog-author {
+.blog-authors {
   font-style: italic;
 }
+.blog-author-link {
+  color: inherit;
+  text-decoration: none;
+}
+.blog-author-link-label {
+  text-decoration: underline;
+}
+.blog-author-avatar {
+  border-radius: 50%;
+  display: inline;
+  vertical-align: middle;
+  width: 1.5em;

```

Comment:
Could we just push the avatar over a tiny bit to match the spacing on both sides? I have no idea if this is the right value (aka it is probably too much) but figured I would leave it up to y'all to get the spacing right.

```suggestion
  width: 1.5em;
  padding: .25em;
```

Reply:
Done 👍 I added a half character of left margin which events things out better. It now looks like this:

![Screenshot 2025-05-28 at 09 32 08](https://github.com/user-attachments/assets/b8867bb7-2aa8-42c6-ac63-8de26c6a69c1)